src/config/env.rs

```diff
@@ -35,6 +35,9 @@ mod tests {
     #[test]
     #[serial]
     fn create_profile_from_empty_env() {
+        env::remove_var(CT_API_KEY);
```

Comment:
Do you have these variables set in your environment, separate from tests? If not, that speaks to a bigger issue to resolve. The point of the `#[serial]` annotation is so all of these tests that adjust the environment run serially, to avoid interleaving writes allow them to clean up after themselves.

Reply:
Yes, I did have the CT_API_KEY set in my environment, since I had been using my terminal window to do live testing.

Reply:
Gotcha. I always ended up using the config file with CLI args overrides to avoid that. CI doesn't hit it either because it always starts clean. This looks good.